addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
```

Comment:
Im not disagreeing, the fact that it isnt one sided is natural.

My take is, if it is such an eye sore, if it really is, then I can change it back. For goodness sake I dont wish that to be the one thing people get stuck on.